packages/amplify-category-auth/src/provider-utils/awscloudformation/index.ts

```diff
@@ -96,6 +96,10 @@ export const updateConfigOnEnvInit = async (context: $TSContext, category: any,
       envSpecificParametersResult = envSpecificParameters;
     } else {
       // succeeded === false | undefined
+      if (cleanupDone) {
+        // returning undefined as nothing to pass here
+        return envSpecificParametersResult;
+      }
```

Comment:
There is more logic involved in updating state files when ```succeeded```  flag is true: 
https://github.com/aws-amplify/amplify-cli/blob/7f50d3f4118a8e2ed749e9fc425536bad1177d3f/packages/amplify-category-auth/src/provider-utils/awscloudformation/import/index.ts#L1198

Reply:
I do not see relation between `succeed` and that code fragment. Don't we just want to skip lines below because resource got removed ?
If so then I'd return `{ succeded: true, resourceRemoved: true }` (i.e. rename that cleanupDone - it's vague and short circuit the flow in `succeed` branch (if we removed orphan successfully why wouldn't this count as success? ) `if (succeded) { if(resourceRemoved) { return; } else {...`

Reply:
I see , I can change this.